esx_service/cli/vmdkops_admin.py

```diff
@@ -46,6 +46,8 @@
 def main():
     log_config.configure()
     kv.init()
+    if not vmdk_ops.is_service_available():
+       sys.exit('Unable to connect to the host-agent on this host, ensure the ESX host agent is running before retrying.')
```

Comment:
Ah, yes. Sorry I should have paid more attention to file names :-)

No more issues then. LGTM